server/seccomp/types.go

```diff
@@ -0,0 +1,93 @@
+package seccomp
+
+// Seccomp represents the config for a seccomp profile for syscall restriction.
+type Seccomp struct {
```

Comment:
okay, I see what you mean. It isn't exactly the same as runtime spec even though some of it is the same.